snapshot-labs/snapshot
# fix: tunnel sentry to avoid ad-blocker

### Issues

The sentry requests are blocked by some ad-blocker. See https://docs.sentry.io/platforms/javascript/troubleshooting/#dealing-with-ad-blockers

Requests will need to go through a tunnel to avoid this.

### Changes 

1. Add the `tunnel` config to sentry setup, which will route all sentry requests to sidekick `/sentry` endpoint

### How to test

1. Open the website
2. Open the NETWORK tab in the dev console
3. Visit some page
4. Look at the NETWORK tab, the sentry request should go to sidekick server instead of sentry, and should return a `200` status code


### Self-review checklist

### Additional notes or considerations
*_(Include any other relevant information or context that may be helpful for the reviewer)_*
